data/fields/dog_check.json

```
@@ -0,0 +1,5 @@
+{
+    "key": "dog",
+    "type": "check",
+    "label": "Suitable or intended for dogs"
```

```suggestion
    "label": "Suitable for dogs"
```

Let remove the "intended" which sounds too much like `dog=designated` to me and we don't provide this value in the field.

Also, the label should always be camel cased
```suggestion
    "label": "Suitable For Dogs"
```

[Example ](https://github.com/openstreetmap/id-tagging-schema/blob/e9f520a613e407421cf96ce4c8e46062a0d9c1d7/data/fields/historic/wreck/visible_at_low_tide.json#L4), [Search ](https://github.com/search?q=repo%3Aopenstreetmap%2Fid-tagging-schema+label%22%3A&type=code&p=4)